src/lib/instanceManager/createInstanceManager.js

```diff
@@ -101,12 +101,12 @@ module.exports = ({
     addMonitor(newMonitor) {
       window.__alloyMonitors.push(newMonitor);
       Object.keys(calledMonitors).forEach((methodName) => {
-        calledMonitors[methodName].forEach((args) => {
-          if (newMonitor[methodName]) {
+        if (newMonitor[methodName]) {
+          calledMonitors[methodName].forEach((args) => {
             newMonitor[methodName](...args);
-            calledMonitors[methodName] = [];
-          }
-        });
+          });
+          calledMonitors[methodName] = [];
```

Comment:
Why blank out the calledMonitors? This makes it so that if you have two events with onConfigure hooks, only one of them will be called.

Reply:
yeah, fixed it.